src/AzureRenderManager/WebApp/Models/Environments/EnvironmentBaseModel.cs

```diff
@@ -20,5 +20,9 @@ public class EnvironmentBaseModel
         public RenderManagerType? RenderManager { get; set; }
 
         public string RenderManagerName => RenderManager.HasValue ? RenderManager.GetDescription() : "Render Manager";
+
+        public string Error { get; set; }
```

Comment:
what's the difference here - should this be like `ErrorTitle`?

Reply:
found it above - this is exception message and the other is the entire exception details

Reply:
Also good idea, but again this feeds through to various places.